modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -537,7 +536,6 @@ definition {
 
 	macro assertObjectActionsNotPresent {
 		AssertElementNotPresent(
-			key_actionActive = "${actionActive}",
```

Comment:
The path has been modified and is only used to see the name, it does not work anymore to see if it is active.